src/repl.rs

```diff
@@ -120,7 +120,10 @@ pub fn repl<P: AsRef<Path>, F: LurkField>(lurk_file: Option<P>) -> Result<()> {
                         if is_meta {
                             repl.state.handle_meta(&mut s, expr, &package, p)?
                         } else {
-                            repl.state.handle_non_meta(&mut s, expr, false)?
+                            if let Err(e) = repl.state.handle_non_meta(&mut s, expr, false) {
+                                println!("Evaluation error: {}", e);
+                            }
+                            continue;
```

Comment:
I think maybe this PR would be done if the only change at all was to add the `continue` here.